fetch-schema.js

```diff
@@ -0,0 +1,20 @@
+/* eslint-disable */
+require('dotenv').config({ path: '.env.local' })
+const exec = require('child_process').exec
+const dataConfig = require('./relay.config')
+const thegraphConfig = require('./relay_thegraph.config')
+/* eslint-enable */
+
+exec(`get-graphql-schema https://api.thegraph.com/subgraphs/name/uniswap/uniswap-v3 > ${thegraphConfig.schema}`)
```

Comment:
can you make this URL and the one below a const at the top of file? also can you use the beta URL or no?

Reply:
This is for thegraph, I don't think there is a beta url